src/pages/SavedWorkflows.js

```diff
@@ -25,10 +26,10 @@ export const SavedWorkflows = ({ runsData }) => {
           columns: [
             {
               size: { basis: 500 },
-              field: 'workflowName',
-              headerRenderer: () => h(Sortable, { sort, field: 'workflowName', onSort: setSort }, ['Workflow Name']),
+              field: 'workflowUrl',
+              headerRenderer: () => h(Sortable, { sort, field: 'workflowUrl', onSort: setSort }, ['Workflow Link']),
               cellRenderer: ({ rowIndex }) => {
-                return h(TextCell, [sortedPreviousRuns[rowIndex].name])
+                return h(TextCell, [sortedPreviousRuns[rowIndex].workflow_url])
```

Comment:
just a small style comment: is this in snake case on purpose, to differentiate it from other places where we use `workflowUrl`? should we use camelCase?

more broadly: have we considered auto-formatting for this repo?

Reply:
A common reason that may be at play here is that JSONs almost always seem to use snake case for keys, and when they are parsed into a codebase that uses camel case they look out of place.